hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/handlers/ContractCallHandler.java

```diff
@@ -55,6 +55,10 @@ public void handle(@NonNull final HandleContext context) throws HandleException
                 .contractCallResult(outcome.result())
                 .contractID(outcome.recipientIdIfCalled())
                 .status(outcome.status());
+
+        if (!outcome.isSuccess()) {
```

Comment:
Great catch, that is also fixed in [this PR](https://github.com/hashgraph/hedera-services/pull/8635) but at the rate I'm getting approvals, you will merge first. 😂